crates/dojo-core/src/world.cairo

```diff
@@ -202,12 +221,25 @@ mod world {
         values: Span<felt252>,
     }
 
+    #[derive(Drop, starknet::Event)]
+    struct StoreSetEntity {
+        model_id: felt252,
+        entity_id: felt252,
+        values: Span<felt252>,
+    }
+
     #[derive(Drop, starknet::Event)]
     struct StoreDelRecord {
         table: felt252,
         keys: Span<felt252>,
     }
 
+    #[derive(Drop, starknet::Event)]
+    struct StoreDeleteEntity {
```

Comment:
For this one @Larkooo, what do you think if the delete always send only the entity id? Like this we keep only one `StoreDelRecord`, which takes the `entity_id` instead of the keys.

We will adjust the macro `delete!` accordingly to compute the hash under the hood. And for the `.delelte` it already works like this.